File: Java Assignments/Day02Assignment/Student.java
package com.sonata;

public class Student {
	int stdid;
	String stdname;
	int stdClass;
	
	Student(int n, String name, int c){
		this.stdid = n;
		this.stdname= name;
		this.stdClass=c;
		}
		Student(){}
	
	public void display()
	{
		System.out.println(stdid);
		System.out.println(stdname);
		System.out.println(stdClass);
	}
	
	public static void main(String args[])
	{
		Student s1 =new Student();
		s1.stdid=101;
		s1.stdname="Lucky";
		s1.stdClass=9;

		s1.display();
		
		Student s2 = new Student(102,"Laya",8);

		s2.display();
		
		
		
	}



}

File: TypeScript/Student.js
"use strict";
exports.__esModule = true;
exports.Student = void 0;
var Student = /** @class */ (function () {
    function Student(i, n) {
        this.stdID = 123;
        this.stdName = "Sujani";
        this.stdID = i;
        this.stdName = n;
    }
    Student.prototype.display = function () {
        console.log(this.stdID);
        console.log(this.stdName);
    };
    return Student;
}());
exports.Student = Student;
var s1 = new Student(124, "lucky");
s1.display();
